lib/mayaUsd/commands/Readme.md

```diff
@@ -0,0 +1,440 @@
+| **Mesh nodes (internal for UsdMaya)**: Internal to Maya; cannot be set using adaptors. |
+| `USD_EmitNormals`                         | bool   | true/false      | UsdMaya uses this attribute to determine if mesh normals should be emitted; by default, without the tag, UsdMaya won't export mesh normals to USD. **Note**: Currently Maya reads/writes face varying normals. This is only valid when the mesh's subdivision scheme is `none` (regular poly mesh), and is ignored otherwise. |
+| **Mesh nodes (UsdGeomMesh attributes)**: These attributes get converted into attributes of the UsdGeomMesh schema. You can use UsdMaya adaptors to author them. Note that these are only the common mesh attributes; you can export any known schema attribute using UsdMaya adaptors. |
+| `USD_ATTR_faceVaryingLinearInterpolation` | string | `none`, `cornersOnly`, `cornersPlus1`, `cornersPlus2`, `boundaries`, `all` | Determines the Face-Varying Interpolation rule. Used for texture mapping/shading purpose. Defaults to `cornersPlus1`. See the [OpenSubdiv documentation](http://graphics.pixar.com/opensubdiv/docs/subdivision_surfaces.html#face-varying-interpolation-rules) for more detail. |
+| `USD_ATTR_interpolateBoundary`            | string | `none`, `edgeAndCorner`, `edgeOnly` | Determines the Boundary Interpolation rule. Valid for Catmull-Clark and Loop subdivision surfaces. Defaults to `edgeAndCorner`. |
+| `USD_ATTR_subdivisionScheme`              | string | `none`, `bilinear`, `catmullClark`, `loop` | Determines the Mesh subdivision scheme. Default can be configured using the `-defaultMeshScheme` export option for meshes without `USD_ATTR_subdivisionScheme` manually specified; we currently default to `catmullClark`. |
+
+
+#### Export Chasers (Advanced)
+
+Export chasers are plugins that run as part of the export and can
+implement prim post-processing that executes immediately after prims
+are written (and/or after animation is written to a prim in time-based
+exports). Chasers are registered with a particular name and can be
+passed argument name/value pairs in an invocation of `MayaUSDExportCommand`.
+There is no "plugin discovery" method here – the developer/user is
+responsible for making sure the chaser is registered.
+
+We provide one such chaser plugin called `AlembicChaser` to try to make
+integrating USD into Alembic-heavy pipelines easier. One of its features
+is that it can export all Maya attributes whose name matches a particular
+prefix (e.g. `ABC_`) as USD attributes by using its `attrprefix` argument.
+Here is an example of what that call to `mayaUSDExport` might look like:
```

Comment:
The mechanism for registering chasers is in the core of mayaUsd, but the Alembic chaser was left behind in the `pxrUsd` plugin:
https://github.com/Autodesk/maya-usd/blob/dev/plugin/pxr/maya/plugin/pxrUsd/alembicChaser.cpp

So all of the Alembic-related sections below should probably be removed.

It might be nice though to keep a generic description and example here of what an invocation of `mayaUSDExport()` looks like when you're using a chaser that takes argument.

Reply:
Would a simple mention that the alembic chaser is given as an example and is actually part of the pxr plugin be enough?

Reply:
Ah, but if the alembic chaser is part of pxr, but the example mention the mayaUSDExport command, that would be a bit weird...

Reply:
(OTOH, maybe any plugin can use the chaser of any other plugin? Then the example would at least work...)

Reply:
Yeah, in the short term, if you wanted to add something that points to the Alembic chaser as an example, and includes a mention that you would need to `cmds.loadPlugin('pxrUsd')` for the example to work, that might be enough for now.

Longer term, I think we'll want to decide what to do with the Alembic chaser. We (Pixar) never used it internally. It was written really early on when we open sourced the plugin to make it a bit easier for external users to try out USD. If people are still making use of it, it should probably get moved to the adsk plugin or some other new alembicUsdExportPlugin or whatever. If it stays in pxrUsd, it'll probably end up going away with the rest of the stuff in there.